config/peerpods/podvm/Dockerfile.podvm-builder.openshift

```diff
@@ -0,0 +1,40 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.5-1736404036
+
+# azure-podvm-image-handler.sh script under /scripts/azure-podvm-image-handler.sh
+# aws-podvm-image-handler.sh script under /scripts/aws-podvm-image-handler.sh
+# sources for cloud-api-adaptor under /src/cloud-api-adaptor
+# The podvm binaries are expected to be under /payload/podvm-binaries.tar.gz
+# Binaries like kubectl, packer and yq under /usr/local/bin will be installed by the scripts
+
+
+LABEL kata_src=https://github.com/openshift/kata-containers
+LABEL kata_src_commit=osc-release
+
+ARG ORG_ID
+ARG ACTIVATION_KEY
+
```

Comment:
About the  subscription code that was here... I'm not familiar on how konflux handles this (maybe @spotlesstofu). 

But, I was wondering if something like this would work:

```
RUN if [[ -n "${KONFLUX_BUILD}" ]]; then \
```

Not sure if the konflux already exposes any variable, but if not, maybe we can configure those variables within the tekton folder. This way we could avoid having two dockerfiles. Wdyt?